src/main/java/seedu/address/model/Dashboard.java

```diff
@@ -156,28 +156,28 @@ public int getNumClosedClients() {
         return totalClosedClients;
     }
 
-    public ObservableList<Reminder> getReminderList() {
+    public ObservableList<Pair<LocalDate, ObservableList<Reminder>>> getDateReminders() {
         updateDashboardIfDirty();
-        return reminderList;
```

Comment:
I think `getReminderList` might be used somewhere in the tests that I've created, have we checked that?